<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExposedServicesReportsSetupDialog</class>
 <widget class="QDialog" name="ExposedServicesReportsSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>218</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>397</width>
    <height>218</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>397</width>
    <height>218</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="1">
    <widget class="QGroupBox" name="grbPeriod">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Период</string>
     </property>
     <widget class="CDateEdit" name="edtBegDate">
      <property name="geometry">
       <rect>
        <x>143</x>
        <y>18</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblEndDate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>42</y>
        <width>129</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Дата &amp;окончания периода</string>
      </property>
      <property name="buddy">
       <cstring>edtEndDate</cstring>
      </property>
     </widget>
     <widget class="CDateEdit" name="edtEndDate">
      <property name="geometry">
       <rect>
        <x>143</x>
        <y>42</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblBegDate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>18</y>
        <width>129</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Дата &amp;начала периода</string>
      </property>
      <property name="buddy">
       <cstring>edtBegDate</cstring>
      </property>
     </widget>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QWidget" name="wgtCommon" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>100</height>
      </size>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>371</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLabel" name="lblContract">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Договор</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblOrgStructure">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Подразделение</string>
         </property>
         <property name="buddy">
          <cstring>cmbOrgStructure</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="COrgStructureComboBox" name="cmbOrgStructure">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="CContractComboBox" name="cmbContract"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblFinance">
         <property name="text">
          <string>Тип финансирования</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="CRBComboBox" name="cmbFinance"/>
       </item>
       <item row="3" column="1">
        <widget class="CRBComboBox" name="cmbEventType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblEventType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Тип события</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CContractComboBox</class>
   <extends>QComboBox</extends>
   <header>Accounting/ContractComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExposedServicesReportsSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExposedServicesReportsSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
